frontend/src/components/P5Wrapper.jsx

```diff
@@ -48,13 +50,13 @@ const P5Wrapper = () => {
     sketch = type.sketch;
     gamemodeDataFilePath = type.filePath;
 
-    const p5Instance = new p5((p) => sketch(p, gamemodeDataFilePath, gameContext.dispatch), sketchRef.current);
+    const p5Instance = new p5((p) => sketch(p, gamemodeDataFilePath, {dispatch: gameContext.gamemodeDispatch, popupVisible: popup.visible}), sketchRef.current);
```

Comment:
Is it possible that maybe instead of passing it into each sketch, we could make show like a dummy sketch on the screen based on the value of popup.visible? Not necessary to change now, maybe make an issue for it, even if we don't plan on doing it, just to discuss it eventually.

Reply:
I have been thinking a lot about making a placeholder for the sketch as it loads, so making a "dummy sketch" could possibly be used for both of these scenarios. I have made an issue for this.